server/src/main/java/org/opensearch/index/merge/MergeStats.java

```diff
@@ -116,6 +126,14 @@ public void add(
         } else {
             this.totalBytesPerSecAutoThrottle += bytesPerSecAutoThrottle;
         }
+        this.add(mergedSegmentWarmerStats);
+    }
+
+    public void add(MergedSegmentWarmerStats warmerStats) {
+        if (this.getWarmerStats() == null) {
```

Comment:
nit - can we directly use the variable in this class at all places instead of the getWarmerStats method